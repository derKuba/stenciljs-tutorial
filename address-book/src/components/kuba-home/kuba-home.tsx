import { Component, Host, h } from "@stencil/core";

@Component({
  tag: "kuba-home",
  styleUrl: "kuba-home.css",
  shadow: true,
})
export class KubaHome {
  render() {
    return (
      <Host>
        <h1>Kuba-Home</h1>

        <kuba-list></kuba-list>
      </Host>
    );
  }
}
