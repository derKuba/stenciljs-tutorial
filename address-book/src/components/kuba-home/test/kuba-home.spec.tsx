import { newSpecPage } from "@stencil/core/testing";
import { KubaHome } from "../kuba-home";

describe("kuba-home", () => {
  it("renders", async () => {
    const page = await newSpecPage({
      components: [KubaHome],
      html: `<kuba-home></kuba-home>`,
    });
    expect(page.root).toMatchSnapshot();
  });
});
