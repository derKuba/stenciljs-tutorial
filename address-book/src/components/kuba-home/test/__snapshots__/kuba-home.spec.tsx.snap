// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kuba-home renders 1`] = `
<kuba-home>
  <mock:shadow-root>
    <h1>
      Kuba-Home
    </h1>
    <kuba-list></kuba-list>
  </mock:shadow-root>
</kuba-home>
`;
